src/react-native.js

```diff
@@ -102,6 +102,7 @@ const ReactNative = {
   PointPropType: require('./propTypes/PointPropType'),
   NavigationExperimental: require('./Libraries/NavigationExperimental'),
 };
+ReactNative.PickerIOS.Item = createMockComponent('PickerIOS.Item');
```

Comment:
This is the code i mentioned last PR, however, it has nothing to do with this PR, I can delete it.